ci/azp-pipeline.yml

```diff
@@ -53,10 +55,12 @@ jobs:
   timeoutInMinutes: 60
   steps:
   - template: install_deps.yml
+  - script: sudo apt-get install -y python-minimal tox
+    displayName: Install Tox
   - checkout: self
     path: 'go/src/github.com/hyperledger/fabric-ca'
     displayName: Checkout Fabric CA Code
   - script: tox -edocs
     displayName: Build Documentation
-  - script: '$(GOPATH)/go/src/github.com/hyperledger/fabric-ca/docs/_build/html'
-    displayName: Publish Documentation
\ No newline at end of file
+  - publish: '$(GOPATH)/src/github.com/hyperledger/fabric-ca/docs/_build/html'
```

Comment:
Why are we referencing GOPATH? Why isn't this relative to the build directory or working directory?

Reply:
Changed to relative to working directory